ocs_ci/ocs/ui/validation_ui.py

```diff
@@ -169,3 +169,49 @@ def verification_ui(self):
         for err in self.err_list:
             logger.error(err)
         assert len(self.err_list) == 0, f"{self.err_list}"
+
+    def check_capacity_breakdown(self, project_name, pod_name):
+        """
+        Check Capacity Breakdown
+
+        Args:
+            project_name (str): The name of the project
+            pod_name (str): The name of pod
+
+        Returns:
+            bool: True if project_name and pod_name exist on capacity_breakdown, False otherwise
+
+        """
+        self.navigate_overview_page()
+        self.choose_expanded_mode(
+            mode=True, locator=self.validation_loc["capacity_breakdown_options"]
+        )
+        self.do_click(self.validation_loc["capacity_breakdown_projects"])
+        self.take_screenshot()
+        res = True
+        sample = TimeoutSampler(
+            timeout=30,
+            sleep=2,
+            func=self.check_element_text,
+            expected_text=project_name,
+        )
+        if not sample.wait_for_func_status(result=True):
+            logger.error(f"The project {project_name} not found on capacity_breakdown")
+            res = False
+
```

Comment:
IMO, at this point, if res==False, you can return it since it doesn't matter if the pod exists or not. the value of res will not change.

Reply:
In my method, I want to check the pod and the project.. and not raise exception if project does not exist..
The function return `True` if project_name and pod_name exist on capacity_breakdown, `False` otherwise

Reply:
so IIUC, if one of pod or project does not exist then we should return False so it should be OK to just return False here if the project does not exist